packages/system/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.10.1"
+  changes:
+    - description: Fix missing ignore_types in filesystem
+      type: bugfix
+      link: https://github.com/elastic/integrations/pull/2679
```

I think this could be considered an enhancement :)
```suggestion
- version: "1.11.0"
  changes:
    - description: Add option to configure ignored filesystem types
      type: enhancement
      link: https://github.com/elastic/integrations/pull/2679
```